python/trifinger_simulation/collision_objects.py

```diff
@@ -145,3 +138,205 @@ def __del__(self):
         # an error, only remove the object if the simulation is still running.
         if pybullet.isConnected(**self._kwargs):
             pybullet.removeBody(self.block, **self._kwargs)
+
+
+class Object:
+    def __init__(
+        self,
+        type="block",
+        position=[0.05, 0.0, 0.0325],
+        orientation=[0, 0, 0, 1],
+        half_size=0.0325,
+        radius=0.0325,
+        height=0.0325,
+        color_rgba=[255, 255, 255, 1],
+        mass=0.08,
+        lateral_friction=1,
+        spinning_friction=0.001,
+        restitution=0,
+        mesh_file_path=None,
+        is_concave=False,
+        **kwargs,
```

Comment:
Does this `kwargs` argument has any other purpose apart from passing the the physicsClientId?  If not, I would rather replace it with an explicit argument for that.